tlsobs-scanner/main.go

```diff
@@ -234,7 +233,8 @@ func scan(scanID int64, cipherscan string) {
 	// launch workers that evaluate the results
 	resChan := make(chan worker.Result)
 	totalWorkers := 0
-	for _, wrkInfo := range worker.AvailableWorkers {
+	for k, wrkInfo := range worker.AvailableWorkers {
+		workerInput.Params, _ = scan.AnalysisParams[k]
```

Comment:
No, it will just pass `nil` to the worker. I thought it was a better idea for the worker to validate params, because they might be optional to begin with.